__mocks__/glob-results.js

```diff
@@ -127,11 +140,40 @@ const packageFiles = () => {
 			'packages/package/failIsTopLevelFile/folder2/file.json',
 			'packages/package/failIsTopLevelFile/folder2/subfolder',
 			'packages/package/failIsTopLevelFile/folder2/subfolder/file.js'
+		],
+		'packages/package/passContext/*': [
+			'packages/package/passContext/required.md',
+			'packages/package/passContext/HISTORY.md'
+		],
+		'packages/package/passContext/brandA/**/*': [
+			'packages/package/passContext/brandA/folder1',
+			'packages/package/passContext/brandA/folder1/file.scss',
+			'packages/package/passContext/brandA/folder1/file.css',
+			'packages/package/passContext/brandA/folder2',
+			'packages/package/passContext/brandA/folder2/file.js',
+			'packages/package/passContext/brandA/folder2/file.spec.js',
+			'packages/package/passContext/brandA/folder2/file.json',
+			'packages/package/passContext/brandA/folder2/subfolder',
+			'packages/package/passContext/brandA/folder2/subfolder/file.js'
+		]
+	};
+};
+
+const folderFiles = () => {
```

Comment:
Is there a reason that these are functions that return objects, rather than just objects?

Reply:
Absolutely no reason at all. Not sure why I've done that!